opentelemetry-operator/chart/crds/crd-opentelemetrycollector.yaml

```diff
@@ -21,7 +21,6 @@ spec:
           namespace: {{ .Release.Namespace }}
           path: /convert
           port: {{ .Values.admissionWebhooks.servicePort }}
-  {{ if .caBundle }}{{ cat "caBundle:" .caBundle | indent 8 }}{{ end }}
```

Comment:
Why is this not needed anymore?

Reply:
We are not using the caBundle right now and this is currently hindering helm to correctly render the chart.

Reply:
Is this planned to be used in the future? 

If so, add an empty directive in your values and the `if` will evaluate to false.

```yaml
opentelemetry-operator
   [...]
   caBundle:
   [...]
```

Reply:
Not sure, but I agree with you that I make sense to make this optional. The reason I removed it was that the line
`  {{ if .caBundle }}{{ cat "caBundle:" .caBundle | indent 8 }}{{ end }}
`
is afaik not valid in helm and the function `cat` throws an error when installing the chart.

Reply:
the source for the CRDs were wrong. The correct source for CRDs is https://github.com/open-telemetry/opentelemetry-operator/tree/main/config/crd/bases